velox/type/parser/CMakeLists.txt

```diff
@@ -23,15 +23,15 @@ bison_target(
 
 flex_target(
   TypeParserScanner TypeParser.ll ${CMAKE_CURRENT_BINARY_DIR}/Scanner.cpp
-  COMPILE_FLAGS "-Cf --prefix=veloxtp")
+  COMPILE_FLAGS "-Cf --prefix=veloxtpdeprecated")
 
 add_flex_bison_dependency(TypeParserScanner TypeParser)
 
 if(VELOX_MONO_LIBRARY)
   add_custom_target(
-    velox_type_parser_gen_src DEPENDS ${BISON_TypeParser_OUTPUTS}
-                                      ${FLEX_TypeParserScanner_OUTPUTS})
-  add_dependencies(velox velox_type_parser_gen_src)
+    velox_type_parser_gen_src_deprecated DEPENDS ${BISON_TypeParser_OUTPUTS}
```

Comment:
Lets leave these targets as is, but change all other instances of velox_type_parser to refer to velox_presto_type_parser.